server/services/monitoringService.js

```diff
@@ -672,17 +668,16 @@ angular.module('ffffng')
                 .tag('nodes', 'delete-offline')
                 .info(
                     'Deleting offline nodes older than ' +
-                    DELETE_OFFLINE_NODES_AFTER_DURATION.amount + ' ' +
-                    DELETE_OFFLINE_NODES_AFTER_DURATION.unit
+                    config.server.pruneTime + ' days'
                 );
 
             Database.all(
                 'SELECT * FROM node_state WHERE state = ? AND last_seen < ?',
                 [
                     'OFFLINE',
                     moment().subtract(
-                        DELETE_OFFLINE_NODES_AFTER_DURATION.amount,
-                        DELETE_OFFLINE_NODES_AFTER_DURATION.unit
+                        config.server.pruneTime * 86400,
+                        "seconds" // use seconds, so decimal values are possible for days
```

Comment:
"use seconds, so decimal values are possible for days" verstehe ich nicht. Wo ist das Problem mit "days"?